recipes/autoregistry/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "autoregistry" %}
+{% set version = "0.8.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/autoregistry-{{ version }}.tar.gz
+  sha256: e9f84e98622d36b946eeea395e74916426546249422354687fb32d894f9bf8c4
+
+build:
+  noarch: python
+  script:
+    # Removing pyproject.toml here to prevent build from using poetry
+    # because currently a dev version of poetry is required
```

Comment:
Why not use the poetry build here?

Reply:
Hummm. We need to fix these: https://github.com/conda-forge/poetry-feedstock/pulls

Reply:
is this going to take long though?

Reply:
@ocepaf, what's the recommendation for next step? Wait for poetry issues to resolve?

Reply:
I guess, wait ;-p

Reply:
Amazingly it turned out not to be a long wait 😄